App/src/pages/RegularGroups.tsx

```diff
@@ -0,0 +1,83 @@
+import { Container, Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+import { useContext, useEffect, useState } from "react";
+import { GetAllGroups, GroupDTO, MultiGroupResponseDTO } from "../services/group";
+import { UserContext } from "../App";
+import { UserDTO } from "../services/user";
+import Header from "../shared/Header";
+
+export default function DisplayGroups() {
```

Comment:
This file should be renamed to match the name of the function

Reply:
Will do

Reply:
done

Reply:
Well actually I changed the name of the function to match the file but same difference